website/addons/dataverse/templates/log_templates.mako

```diff
@@ -1,27 +1,27 @@
 <script type="text/html" id="dataverse_file_added">
 added file
-<a class="overflow log-file-link" data-bind="attr: {href: params.path}, text: params.filename"></a> to
-Dataverse study <span class="overflow">{{ params.study }}</span>
+<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect, text: params.filename"></a> to
+Dataverse dataset <span class="overflow">{{ params.dataset }}</span>
 in {{ nodeType }}
 <a class="log-node-title-link overflow" data-bind="attr: {href: nodeUrl}">{{ nodeTitle }}</a>
 </script>
 
 <script type="text/html" id="dataverse_file_removed">
 removed file <span class="overflow">{{ params.filename }}</span> from
-Dataverse study <span class="overflow">{{ params.study }}</span>
+Dataverse dataset <span class="overflow">{{ params.dataset }}</span>
 in {{ nodeType }}
 <a class="log-node-title-link overflow" data-bind="attr: {href: nodeUrl}">{{ nodeTitle }}</a>
 </script>
 
-<script type="text/html" id="dataverse_study_linked">
```

Comment:
I think we'll need to keep these log templates in so that old logs are rendered correctly.